contents/tutorials/css-selectors-for-actions.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Creating actions using CSS selectors
+sidebar: Docs
+showTitle: true
+author: ['hophog']
+date: 2022-07-06
+featuredImage: ../images/tutorials/banners/css-selectors-for-actions.png
+topics: ["actions", "CSS selectors", "data attributes"]
+---
+
+_Estimated reading time: 7 minutes_ ☕
+
+PostHog supports different methods for creating [actions](/docs/user-guides/actions): autocapturing, using [Toolbar](/docs/user-guides/toolbar) and via CSS selectors. 
+
+As the name suggests, **autocapturing** is the easiest option, because users don't have to do anything - actions are identified and captured automatically. While this is really nice, the amount of events generated this way can be overwhelming for a targeted analysis and just not meaningfull enough.
```

Comment:
```suggestion
As the name suggests, [**autocapturing** is the easiest option](/blog/is-autocapture-still-bad), because users don't have to do anything - actions are identified and captured automatically. While this is really nice, the amount of events generated this way can sometimes be overwhelming for a targeted analysis.
```